### ISSUE TITLE
Tag ID contains negative number

### ISSUE THREAD

**Issue by gjuchault** [RoopeHakulinen/cordova-plugin-mifare-ultralight#2]
Hi,

My NFC Card has id : `04416DF27D5780`

If I do:

```js
document.addEventListener('mifareTagDiscovered', (tag) => {
    console.log(tag.tag);
})
```

I get: `[4, 65, 109, -14, 125, 87, -128]`

Which makes: `04416D-E7D57-80`

I've added `Log.d(TAG, bytesToHex(tag.getId()));` to `fireTagEvent` to check what id you start with and I have in adb logcat:

```02-19 14:08:51.086 25943 26679 D CordovaPluginMifareUltralight: 04416DF27D5780```

This means that either `byteArrayToJSON` or `MessageFormat.format` has something wrong. Maybe endianness ?
Thanks

**Comment by RoopeHakulinen** [comment]
Hey @gjuchault and thank you for your report. I can't promise to take a look at this for a while now. If you find a solution by forking the repo and playing around with it, please let me know and I'm happy to fix/merge. 

I checked my use case in the project I created this for and it turned out I'm not using the `tag` by myself and thus it is highly possible that it doesn't work as intended.

**Comment by gjuchault** [comment]
@RoopeHakulinen I've submitted the PR :) #3

**Comment by RoopeHakulinen** [comment]
Awesome, thank you so much @gjuchault ! Just released 1.0.2. Please let me know whether it works better or not :)
